libevmasm/Assembly.cpp

```diff
@@ -152,8 +152,8 @@ class Functionalizer
 		if (!m_location.isValid())
 			return;
 		m_out << m_prefix << "    /*";
-		if (m_location.source)
-			m_out << " \"" + m_location.source->name() + "\"";
+		if (m_location.sourceName)
+			m_out << " " + escapeAndQuoteYulString(*m_location.sourceName);
```

Comment:
@cameel you see, it is not only for Yul 🤓

Reply:
This code wasn't here when I was creating my PR :)
I should have kept the version with parameter for selecting between languages.

But still, even though it still works for Solidity, it specifically avoids escapes that are invalid in Yul so I think that the name is adequate.

Reply:
Which escapes are invalid in Yul but valid in Solidity?

Reply:
Looks like only unicode escapes.

Reply:
Ah, you're right! Now I see what you meant :) I was convinced that `\b`, `\v` and `\f` are forbidden only in Yul but I just checked and looks like they're forbidden in Solidity too. In that case I'll rename it back to `escapeAndQuoteYulString()`.